File: C++/1009.cpp
// André Nascimento
// GitHub: github.com/AndreNasci
// Linkedin: linkedin.com/in/andre-nascimento-a01095185

#include <bits/stdc++.h>

using namespace std;

int main()
{
    char name[255];
    cin >> name;

    double salary, sales, total;
    cin >> salary >> sales;

    total = salary + 0.15 * sales;

    cout << "TOTAL = R$ " << fixed << setprecision(2) << total << endl;
    return 0;
}

File: Python/1009.py
"""
André Nascimento
GitHub: github.com/AndreNasci
Linkedin: linkedin.com/in/andre-nasci/
"""

seller_name = input()
fixed_salary = float(input())
month_sales = float(input())

final_salary = fixed_salary + month_sales * 0.15

print(f"TOTAL = R$ {final_salary:.2f}")